packages/shared/src/components/cards/SharePostCard.tsx

```diff
@@ -67,10 +73,11 @@ export const SharePostCard = forwardRef(function SharePostCard(
         title={post.title}
         onHeightChange={onSharedPostTextHeightChange}
       />
-      <Container ref={containerRef}>
+      <Container ref={containerRef} className="justify-end min-h-[0]">
```

Comment:
We'll just have to be careful this doesn't introduce edge-cases for other implementations.

Reply:
This is just used in shared post card, so shouldn't be messing much with other cards... I could test with the data I had and created locally, but could check this if we place it in preview

Reply:
should be ok as soon as we update the branches with main